rpc/tendermint/core/types/responses_util.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015, 2016 Eris Industries (UK) Ltd.
+// This file is part of Eris-RT
+
+// Eris-RT is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Eris-RT is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Eris-RT.  If not, see <http://www.gnu.org/licenses/>.
+
+package core_types
+
+import (
+	"github.com/eris-ltd/eris-db/core/types"
+)
+
+// UnwrapResultDumpStorage does a deep copy to remove /rpc/tendermint/core/types
+// and expose /core/types instead.  This is largely an artefact to be removed once
+// go-wire and go-rpc are deprecated.
+// This is not an efficient code, especially given Storage can be big.
+func UnwrapResultDumpStorage(result *ResultDumpStorage) (*types.Storage) {
```

Comment:
I'm fine with this. We are going to have to do a lot of type mappings (one-way or otherwise) so we might want to put some machinery around it. Ultimately if we are going to have our own types fed from other systems we need to create our object, particularly if they are coming from another process. So I don't see that it's so inefficient. We can replace blind mappings with more selective ones as we go.